src/components/DebatecoreFooter.tsx

```diff
@@ -0,0 +1,71 @@
+import { useLang } from "@/lib/useLang";
+import { Lexend } from "next/font/google";
+
+const lexend = Lexend({ subsets: ["latin"] });
+
+type footerlink = {
+  text: string;
+  href: string;
+};
+
+const DebatecoreFooter = (props: {}) => {
+  const boldfont = lexend.className;
+
+  const footerlinks: footerlink[] = [
+    // {
+    //   text: useLang("footerAbout"),
+    //   href: "/about",
+    // },
+    // {
+    //   text: useLang("footerMission"),
+    //   href: "https://debateco.re/about",
+    // },
+    // {
+    //   text: useLang("footerSendFeedback"),
+    //   href: "https://debateco.re/sendfeedback?from=toolsfooter",
+    // },
```

Comment:
Please remove commented code. Other than that, I once again suggest to include the About page.
```suggestion
    {
      text: useLang("footerAbout"),
      href: "/about",
    },
```